lib/active_merchant/billing/gateways/pin.rb

```diff
@@ -81,6 +81,11 @@ def void(token, options = {})
         commit(:put, "charges/#{CGI.escape(token)}/void", {}, options)
       end
 
+      # Verify a previously authorized charge.
+      def verify(session_token, options = {})
+        commit(:get, "/charges/verify?session_token=#{session_token}", nil, options)
+      end
```

Comment:
`verify` is a reserved action that attempts to verify if a payment method is ready for purchase. It is meant to be a low amount auth & void or 0 amount auth. This does not seem to be doing this so could we change the name of this method? Additionally I don't see this method being called anywhere, am I missing something?

Reply:
@hudakh perhaps we could go with `verify_3ds` or `verify_three_d_secure` here?

Reply:
@aenand, the method is called by the apps using the Pin gateway to verify that the charge was successful. I have renamed as per @AMHOL's suggestion.